src/lib/validation.js

```diff
@@ -0,0 +1,31 @@
+export function validateReviewFormValues(formValues) {
```

Comment:
I’m not sure I understand what this function does. The name suggests it validates some data, but the code seems to imply it actually submits it to the API. Could you clarify please?

Happy to jump on a quick call if you prefer.

Reply:
Good point! After a quick test it seems to work the same the other way. I’d have to make more tests, but it could improve performance a bit.

Reply:
Thanks for testing! No need to fix it now, the arrays aren’t big enough to cause any performance issue now anyway, but something we need to keep in mind.